<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceFilterDialog</class>
 <widget class="QDialog" name="ServiceFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Фильтр услуг</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="1">
    <widget class="CRBComboBox" name="cmbServiceGroup"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Примечение содержит</string>
     </property>
     <property name="buddy">
      <cstring>edtNote</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblServiceGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Группа</string>
     </property>
     <property name="buddy">
      <cstring>cmbServiceGroup</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="chkEIS">
     <property name="text">
      <string>Унаследовано из ЕИС</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblClass">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Класс</string>
     </property>
     <property name="buddy">
      <cstring>cmbClass</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="edtNote"/>
   </item>
   <item row="2" column="1">
    <widget class="CRBComboBox" name="cmbType"/>
   </item>
   <item row="12" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="CDateEdit" name="edtBegDate"/>
     </item>
     <item>
      <widget class="QLabel" name="lblTo">
       <property name="text">
        <string>по</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>edtEndDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDate"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="lblCreatePeriod">
     <property name="text">
      <string>Добавлено в период с</string>
     </property>
     <property name="buddy">
      <cstring>edtCreatePeriodBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Тип</string>
     </property>
     <property name="buddy">
      <cstring>cmbType</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Название содержит</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="chkNomenclature">
     <property name="text">
      <string>Унаследовано из номенклатуры</string>
     </property>
     <property name="tristate">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="3" column="1">
    <widget class="CRBComboBox" name="cmbClass"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Код начинается &amp;с</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblSection">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>&amp;Раздел</string>
     </property>
     <property name="buddy">
      <cstring>cmbSection</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="1" column="1">
    <widget class="CRBComboBox" name="cmbSection"/>
   </item>
   <item row="15" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblPeriod">
     <property name="text">
      <string>Период &amp;c</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <layout class="QHBoxLayout" name="horizontalLayoutCreatePeriod">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="CDateEdit" name="edtCreatePeriodBegDate"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCreatePeriodTo">
       <property name="text">
        <string>по</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>edtCreatePeriodEndDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtCreatePeriodEndDate"/>
     </item>
     <item>
      <spacer name="horizontalSpacerCreatePeriod">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbServiceGroup</tabstop>
  <tabstop>cmbSection</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>cmbClass</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtNote</tabstop>
  <tabstop>chkEIS</tabstop>
  <tabstop>chkNomenclature</tabstop>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtCreatePeriodBegDate</tabstop>
  <tabstop>edtCreatePeriodEndDate</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ServiceFilterDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ServiceFilterDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
